samples/subsys/tracing/sample.yaml

```diff
@@ -28,7 +28,7 @@ tests:
       - mimxrt1064_evk
     integration_platforms:
       - nrf52840dk/nrf52840
-    extra_args: CONF_FILE="prj_sysview.conf"
```

Comment:
with removal of this file, it is not possible to build this sample with sysview with snippets similar to all others. I suggest adding this to the documentation and describe how to build this on the command line with snippets.